engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -24,7 +24,10 @@ trait WorkflowManagerSystem {
   def allowedBackends: List[String] = ConfigFactory.load.getConfig("backend").getStringList("backendsAllowed").toList
   def defaultBackend: String = ConfigFactory.load.getConfig("backend").getString("defaultBackend")
 
+  val isServerMode = false
+
   CromwellBackend.initBackends(allowedBackends, defaultBackend, actorSystem)
+
   // For now there's only one WorkflowManagerActor so no need to dynamically name it
-  lazy val workflowManagerActor = actorSystem.actorOf(WorkflowManagerActor.props(), "WorkflowManagerActor")
+  lazy val workflowManagerActor = actorSystem.actorOf(WorkflowManagerActor.props(isServerMode), "WorkflowManagerActor")
```

Comment:
If `isServerMode` is only used in one place it would probably be preferable to use keyword arguments

```scala
WorkflowManagerActor.props(isServerMode = false)
```

also this will keep @geoffjentry from unleashing his N > 1 argument against constants 😛